### PR TITLE
home page

### PR DESCRIPTION
**Pull request by Winter-G**

### CODE REVIEW

File: src/app/App.jsx [hunk 1]
```diff
@@ -11,3 +11,6 @@ function App() {
 }
 
 export default App
+
```
Comment (rush144) [inline src/app/App.jsx]: remove unused lines

File: src/pages/HomePage/HomePage.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      flexDirection: 'column',
+    }}
+  >
+    <AppBar position="static" sx={{ bgcolor: 'rgba(0, 0, 0, 0.7)' }}>
+      <Toolbar>
+        <Typography variant="h6" sx={{ flexGrow: 1 }}>
+          Group Project
+        </Typography>
+
+        {/* Navigation links */}
```
Comment (rush144) [inline src/pages/HomePage/HomePage.jsx]: remove comment line

File: src/app/App.jsx (hunk 1 above)
Comment (Winter-G) [reply src/app/App.jsx]: done

File: src/pages/HomePage/HomePage.jsx (hunk 2 above)
Comment (Winter-G) [reply src/pages/HomePage/HomePage.jsx]: done

**Pull request merged**